Modules/TPC/src/CheckOfSlices.cxx

```diff
@@ -98,6 +99,15 @@ void CheckOfSlices::configure()
     }
     mExpectedPhysicsValue = common::getFromConfig<double>(mCustomParameters, "expectedPhysicsValue", 1);
   }
+
+  std::string maskingValues = common::getFromConfig<string>(mCustomParameters, "maskedPoints", "");
+  if (maskingValues != "") {
+    std::stringstream ss(maskingValues);
+    std::string point;
+    while (std::getline(ss, point, ',')) {
+      mMaskedPoints.push_back(std::stoi(point));
+    }
+  }
```

Comment:
You could
```
#include "Algorithm/RangeTokenizer.h"
```
and use
```
mMaskedPoints = RangeTokenizer::tokenize<int>(maskingValues);
```

Reply:
Thanks for the nice suggestion! I implemented it